File: Selenium/Activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver import ActionChains
with webdriver.Chrome() as driver:
    action = ActionChains(driver)
    driver.get("https://training-support.net/webelements/keyboard-events")
    print("Web age title - " + driver.title)
    action.send_keys("We are taking FST training").perform()
    src = driver.find_element(By.CSS_SELECTOR, "h1.mt-3")
    print("Confirmation message - " + src.text);	
    driver.quit()

File: Selenium/Activity9.java
package Selenium;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity9 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		Actions action = new Actions(driver);
		driver.manage().window().maximize();
		driver.get("https://training-support.net/webelements/keyboard-events");
		System.out.println("Page Title is : " + driver.getTitle());
		action.sendKeys("We are taking FST training").build().perform();
		WebElement message = driver.findElement(By.cssSelector("h1.mt-3"));
		System.out.println("Entered Text is - " + message.getText());	
		driver.quit();
	}

}
